modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusStateUpdater.scala

```diff
@@ -223,12 +235,11 @@ object ConsensusStateUpdater {
                           hash <- artifact.hashF
                           effect = gossip.spread(ConsensusPeerDeclaration(state.key, Proposal(hash, facilitatorsHash))) *>
                             gossip.spreadCommon(ConsensusArtifact(state.key, artifact))
-                          newState =
+                          newState: ConsensusState[Key, Artifact, Context] =
```

Comment:
Usually, we tend to not declare an explicit type for local variables. I'm fine with adding it sometimes, but here it doesn't seem to be necessary in my opinion as there's no ambiguity based on the variable name. Also in other pattern-match cases (like `CollectingProposals` and `CollectingSignatures`) it wasn't added.

Reply:
most likely it was used before like `newState: Type = ???` and implementation was added later. I'll remove the type